booking-app/components/src/client/routes/booking/formPages/BookingFormConfirmationPage.tsx

```diff
@@ -52,6 +52,49 @@ export default function BookingFormConfirmationPage({ formContext }: Props) {
     );
   }
 
+  if (submitting === "error") {
+    return (
+      <Centered>
+        <Box
+          sx={{
+            position: "absolute",
+            top: "calc(50% - 40px)",
+            left: "50%",
+            transform: "translate(-50%, -100%)",
+          }}
+        >
+          <Typography variant="h3" lineHeight="1.55rem">
+            ⚠️
+          </Typography>
+        </Box>
+        <Typography variant="h6" sx={{ padding: 3, textAlign: "center" }}>
+          This time slot is no longer available.
```

Comment:
@nimanns 
The reason for the error isn’t just the time slot. Can you get the error reason from variables?